src/more-infos/more-info-camera.html

```diff
@@ -29,21 +29,26 @@
     stateObj: {
       type: Object,
     },
+
+    isVisible: {
+      type: Boolean,
+      value: true,
+    },
   },
 
   imageLoaded: function () {
     this.fire('iron-resize');
   },
 
-  computeCameraImageUrl: function (hass, stateObj) {
+  computeCameraImageUrl: function (hass, stateObj, isVisible) {
     if (hass.demo) {
       return '/demo/webcam.jpg';
-    } else if (stateObj) {
+    } else if (stateObj && isVisible) {
```

Comment:
The call is "window.hassUtil.dynamicContentUpdater(this, null, { isVisible: false })"
And dynamicContentUpdater doesn't clear attributes which are not passed, so just isVisible is changed to false but, stateObj keeps its previous value.

At first I though just to pass stateObj:null instead of adding a new attribute, but some other classes expect stateObj to be non null.